app/scripts/modules/amazon/src/pipeline/stages/deployCloudFormation/evaluateCloudFormationChangeSetExecutionDetails.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+
+import { IExecutionDetailsSectionProps } from 'core/pipeline';
+import { IStage } from 'core/domain';
+import { ExecutionDetailsSection } from '@spinnaker/core';
+import {
+  EvaluateCloudFormationChangeSetExecutionApproval,
+  IEvaluateCloudFormationChangeSetExecutionApprovalState,
+} from './evaluateCloudFormationChangeSetExecutionApproval';
+
+export interface IEvaluateCloudFormationChangeSetExecutionDetailsState {
+  parentDeployStage: IStage;
+}
+
+export class EvaluateCloudFormationChangeSetExecutionDetails extends React.Component<
+  IExecutionDetailsSectionProps,
+  IEvaluateCloudFormationChangeSetExecutionApprovalState
```

Comment:
this component never calls `setState()` so shouldn't have a state interface declared in this spot.